src/components/Nav/Nav.tsx

```diff
@@ -196,6 +194,14 @@ export class Nav extends React.Component<INavProps, INavState> implements INav {
     ev.preventDefault();
     ev.stopPropagation();
   }
+
+  private _onNavLinkClicked(link: INavLink, ev: React.MouseEvent): void {
+    this._selectedKey = link.key;
+
+    if (link.onClick) {
+      link.onClick();
```

Comment:
So 2 suggestions:

1. A quick fix: you're not passing any information into the click handler. The click handler should have something like:

ev?: React.MouseEvent, item?: INavLink 

...in that order. And you should pass them in.

2. We really should just have a separate NavLink component that basically is a subclass of a button, because we're seeing tons of inconsistencies across commanding components. You don't need to do this here, but be aware that using "bind" in components is a bad practice and the only reason we are doing it here is because we don't have a subcomponent representing the item. (If we did you could just refer to this.props.item.)